docs/source/concepts/AbsorptionAndMultipleScattering.rst

```diff
@@ -173,6 +173,25 @@ For elastic scattering, :math:`\lambda_1` = :math:`\lambda_2` = :math:`\lambda`
 
     A_{elastic} = \frac{1}{V} \int_{V} exp \left[ -\mu (\lambda) \left( l_1 + l_2 \right) \right] dV
 
+Gauge Volume
+############
+
+The gauge volume refers to the region of the sample that is both illuminated by the incident neutron beam and visible to the detector. Only neutrons that are scattered within this volume and reach the detector contribute to the measured signal. Therefore, accurate absorption corrections must account for the size and shape of the gauge volume to avoid over- or under-estimating path lengths through absorbing material.
+
+In many experimental setups, especially for complex geometries or partial beam coverage, the gauge volume may be significantly smaller than the full sample volume. To accommodate this, Mantid provides multiple methods for defining the gauge volume. These methods are prioritized in the following order:
```

Comment:
The gauge volume refers to the region of the sample that is both illuminated by the incident neutron beam and visible to the detector. Only neutrons that are scattered within this volume and reach the detector contribute to the measured signal. Therefore, accurate absorption corrections must account for the size and shape of the gauge volume to avoid over- or under-estimating the absorption evaluation.

In normal experimental setups, the neutron beam only partially covers the sample and therefore the gauge volume has to be defined to be smaller than the sample volume. To accommodate this, Mantid provides multiple methods for defining the gauge volume. These methods are prioritized in the following order: